Assets/Scripts/Level generation scripts/NavigationGraph.cs

```diff
@@ -0,0 +1,331 @@
+﻿using System.Linq;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Node
+{
+    public Vector3 position;
+    public List<Link> links;
```

Comment:
These two variables could probably be private and provide getters and setters. I don't see them being called outside of this class very often